stable/mission-control/templates/postgresql-secrets.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.postgresql.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "mission-control.fullname" . }}-db-cred
+  labels:
+    app: {{ template "mission-control.name" . }}
+    chart: {{ template "mission-control.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+type: Opaque
+data:
+  {{- if .Values.postgresql.db.jfisPassword }}
+  jfisPassword: {{ .Values.postgresql.db.jfisPassword | b64enc | quote }}
+  {{- else }}
+  jfisPassword: {{ randAlphaNum 10 | b64enc | quote }}
+  {{- end }}
+  {{- if .Values.postgresql.db.jfscPassword }}
+  jfscPassword: {{ .Values.postgresql.db.jfscPassword | b64enc | quote }}
+  {{- else }}
+  jfscPassword: {{ randAlphaNum 10 | b64enc | quote }}
+  {{- end }}
+  {{- if .Values.postgresql.db.jfexPassword }}
+  jfexPassword: {{ .Values.postgresql.db.jfexPassword | b64enc | quote }}
+  {{- else }}
+  jfexPassword: {{ randAlphaNum 10 | b64enc | quote }}
```

Comment:
Why are all of these here if you removed it from the values?

Reply:
These are for internal postgres ~ postgresql.enabled=true, removed only for external database password